app/src/main/res/layout-land/ongoing_topic_list_fragment.xml

```diff
@@ -52,8 +52,10 @@
         android:layout_height="match_parent"
         android:clipToPadding="false"
         android:overScrollMode="never"
+        android:paddingStart="56dp"
+        android:paddingEnd="56dp"
         android:paddingTop="28dp"
-        android:paddingBottom="108dp"
+        android:paddingBottom="92dp"
```

Comment:
This will be `88dp` only as the bottom margin of the item is `12dp` and `100-12=88`

Reply:
Got it.

Reply:
Fixed in new PR #1113.

Reply:
Done.